### DISCUSSION TITLE
Preview problems

### DISCUSSION THREAD

**Discussion by raddie361** [brackets-cont/brackets#124] [Q&A]
Hi! New to coding, but I'm pretty sure the code is right (like 75% sure the code is right), but when I preview it, it's wrong. Any changes I make to the CSS don't reflect in the preview (after saving the changes). Any ideas why these changes aren't updating?

**Comment by charlypa** [comment]
@raddie361 
What's your Operating system and browser used for your coding?

**Reply by raddie361** [reply]
Windows and Chrome (all up to date)

**Reply by charlypa** [reply]
@raddie361 
Apologies some how I missed this thread
Are you still facing the issue?

**Reply by raddie361** [reply]
I went back to check it when I saw your comment and noticed the problem! Brackets didn't like the quotation marks. I have to copy the code over from a pdf, and that makes the quotation marks very janky. Thank you! Sorry, I'm an idiot.

**Reply by charlypa** [reply]
@raddie361 
Don't worry. All questions are good questions. Happy coding :). 
Keep using brackets.
